tgtg_scanner/notifiers/smtp.py

```diff
@@ -79,29 +96,48 @@ def _stay_connected(self) -> None:
         if status != 250:
             self._connect()
 
-    def _send_mail(self, subject: str, html: str) -> None:
+    def _send_mail(self, subject: str, html: str, item_id: int) -> None:
```

Comment:
The item_id should already be of type `str`. But this is an issue with bad typing in models/item.py and a separate task for further refinement.
No change required.